src/main/java/list_excercises/service/shuffle.java

```diff
@@ -0,0 +1,31 @@
+package list_excercises.service;
+import java.util.ArrayList;
+import java.util.Random;
+
+public class shuffle {
+
+    public ArrayList<String> shuffleArray(ArrayList<String> arr) {
```

Comment:
I've seen that it's been introduced like this in the course, but it's a good practice to declare the interface and not the implementation itself, so it could be List<String> as the return value, and as an argument and whenever you declare it ;)

Here's the article: 
https://stackoverflow.com/questions/12321177/arraylist-or-list-declaration-in-java